MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,530 @@
+
+1. HeaderToolbarItem
+2. FooterToolbarItem
+3. SubItems
+
+## Adding Toolbar Items to the Header Toolbar
+
+The `ImageEditorToolbarItem` is placed on the top of the image editor, and you can customize the header toolbar item using the `View` and `Name` properties:
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image4.png");
+        var headerToolbar = imageEditor.Toolbars[0].ToolbarItems[0] as ImageEditorToolbarGroupItem;
+        headerToolbar.Items.Add(new ImageEditorToolbarItem { Name = "Share", View = new Image { Source = ImageSource.FromFile("share.png") }});
+
+{% endhighlight %}
+
+## Adding Toolbar Items to the Footer Toolbar
+
+The `ImageEditorToolbarItem` is placed on the bottom of the image editor, and you can customize the footer toolbar item using the `View` and `Name` properties.
+
+Refer to the following code snippet to customize footer toolbar item.
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image4.png");
+        var footerToolbar = imageEditor.Toolbars[1];
+        footerToolbar.ToolbarItems.Add(new ImageEditorToolbarItem { Name = "Share", View = new Image { Source = ImageSource.FromFile("image4.png") }});
+
+{% endhighlight %}
+
+## Adding SubItems to the FooterToolbarItem
+
+The `SubItems` is only applicable for `FooterToolbarItem`, and it represents grouped action of respective footer toolbar item. The SubItems will be placed above the footer toolbar item layout, and you can also customize the appearance of sub items as main toolbar items.
+
+Refer to the following code snippet to customize sub items of footer toolbar item.
+
+{% highlight C# %}
+
+ 	editor.ToolbarSettings.ToolbarItems.Add(new FooterToolbarItem()
+            {
+                Text = "More",
+                Icon = ImageSource.FromResource("ImageEditor.Image.more.png"),
+                SubItems = new ObservableCollection<ToolbarItem>()
+                {
+                   new ToolbarItem() {
+                Icon = ImageSource.FromResource("ImageEditor.Image.download.png")
+                },
+                   new ToolbarItem() {
+                Icon = ImageSource.FromResource("ImageEditor.Image.share.png")
+                } }
+            });
+
+{% endhighlight %}
+
+N> You can remove the existing toolbar items `Name` from image editor toolbarItems collection based on the index value and change the icon and text values dynamically for any of the already added toolbar item based on the index as shown in the following code snippet.
+
+{% highlight C# %}
+
+editor.ToolbarSettings.ToolbarItems[5].Text = "new item";
+editor.ToolbarSettings.ToolbarItems[3].Icon = ImageSource.FromResource("ImageEditor.Image.jpg");
+
+{% endhighlight %}
+
+## ToolbarItemSelected event 
```

Comment:
Default toolbar item action can be restricted by setting e.cancel to true

Reply:
Added above content